src/lib/expansion/expansion-panel-header.scss

```diff
@@ -29,9 +29,13 @@
 
 .mat-expansion-panel-header-title,
 .mat-expansion-panel-header-description {
-  display: flex;
+  display: block;
```

Comment:
Should this be inline block, or are we relying on flex to handle the non-blocking?

Reply:
We need it to be a block for the ellipsis overflow to work.